main.sh

```diff
@@ -31,12 +35,15 @@ done
 function poll_status {
   while true;
   do
-    status=$(curl "$url" -s | jq '.status');
+    status=$(curl "$url" -s -H "Authorization: Basic "$token"" | jq '.status'); #Not sure how to implement the token here
```

Comment:
You don't have to use quotes around `$token` and definitely don't want to do `""` as the script will fall apart.

You can see some examples here, but in general, I think you've done it right (apart from the double double quotes of course 😉)
https://www.misterpki.com/curl-authentication/

Reply:
Okay thanks, I thought that was wrong and tried putting single quotes around token but it didn't like that haha.